src/containers/trends/TrendsSVGContainer.js

```diff
@@ -48,9 +48,8 @@ import { THREE_HRS } from '../../utils/datetime';
 import Background from '../../components/trends/common/Background';
 import CBGSlicesContainer from './CBGSlicesContainer';
 import SMBGsByDateContainer from './SMBGsByDateContainer';
-import SMBGRangeAvgAnimationContainer from './SMBGRangeAvgAnimationContainer';
-import SMBGAvg from '../../components/trends/smbg/SMBGAvg';
```

Comment:
Yeah this is still an open question after our meeting last Thursday, and we're going to revisit once the rest of the new feature work is ready for design review. With the styling changes the average was really distracting, and the info appears on hover, so as a possible temporary or interim solution we've decided to just remove it. I thought I had updated the PR description to make a note of this, but I see I didn't. Sorry! Must have forgot.

I've actually made a notes of the commits to revert if/when we bring the average back: 316f24b985d0401554dab0d30e741ec1346810a5 here in viz and 72395fa4eaada1e0c073d435e5ea9370055568cc in blip.